addons/project_timesheet_holidays/models/hr_employee.py

```diff
@@ -16,22 +16,23 @@ def create(self, vals_list):
         # We need to create timesheet entries for the global time off that are already created
         # and are planned for after this employee creation date
 
-        self._create_future_public_holidays_timesheets(employees)
+        self.with_context(allowed_company_ids=employees.company_id.ids)._create_future_public_holidays_timesheets(employees)
```

Comment:
indentation seems off on my end :nerd_face:

Reply:
I wrote that line in 2 to improve the readability. 😁